.babelrc

```diff
@@ -4,6 +4,12 @@
     "@babel/typescript"
   ],
   "plugins": [
-    "@babel/proposal-class-properties"
+    "@babel/proposal-class-properties",
+    ["@babel/plugin-transform-runtime",
```

Comment:
@DianaVargasOrellana I don't think we will need this plugin since we are using TypeScript, I think this plugin is used to have the Ecma6 built-ins like Promise, Map, etc. in a javascript environment, but, we are using TypeScript, so those built-ins are managed the TS compiler.